### DISCUSSION TITLE
How to enable Stereo Tool web interface?

### DISCUSSION THREAD

**Discussion by mishcazzulani** [AzuraCast/AzuraCast#6701] [Q&A] [answered]
Hi I've installed the Stereo Tool plugin but I can't figure out how to connect to the web interface: I enabled it in the config file and edited the whitelist, but I still cannot reach it.

**Comment by mishcazzulani** [comment, marked as answer]
Ok I've got it: you have to edit /var/azuracast/docker-compose.yml and open the stereo tool port.

**Comment by netbladenl** [comment]
Hi, Do you have the right settings to do this? I really would like to have the webinterface enabled
services:
    web:
        container_name: azuracast
        image: 'ghcr.io/azuracast/azuracast:${AZURACAST_VERSION:-latest}'
        labels:
            - com.centurylinklabs.watchtower.scope=azuracast
        ports:
            - '${AZURACAST_HTTP_PORT:-80}:80'
            - '${AZURACAST_HTTPS_PORT:-443}:443'
            - '${AZURACAST_SFTP_PORT:-2022}:2022'
            -  '${AZURACAST_Stereotool:-8080}:8080'
I added the last line to the compose file, will this be enough? I cannot acces the web interface with this settings :(

**Reply by mishcazzulani** [reply]
Hi, you also have to edit the .sts file at these lines:

Enable web interface=1
Port=8080
Whitelist=X.X.X.X

where xxxx is the public ip address of the pc you're using to connect to your server.

**Reply by netbladenl** [reply]
I used port 8060 with a whitelist of my local subnet which works, I cannot use 8080 but it works fine with me now. Finally I can edit settings on the fly <3

**Comment by wppronto** [comment]
Another way to tweak settings: Download the Stereo Tool app for your platform (PC, Mac, etc.), tweak settings and save. You can also import an existing preset to tweak.

**Reply by wppronto** [reply]
I did the above for the web interface and I still get a "could not connect to server" error for port 8080. I did add the line to the Docker config file and also the lines to the .sts file. After that didn't work, to be sure I also made sure 8080 was open in the Linode firewall. I'm not sure what else I'm missing.
